web/src/env.mjs

```diff
@@ -71,6 +71,11 @@ export const env = createEnv({
     AUTH_COGNITO_CLIENT_SECRET: z.string().optional(),
     AUTH_COGNITO_ISSUER: z.string().url().optional(),
     AUTH_COGNITO_ALLOW_ACCOUNT_LINKING: z.enum(["true", "false"]).optional(),
+    AUTH_CUSTOM_CLIENT_ID: z.string().optional(),
+    AUTH_CUSTOM_CLIENT_SECRET: z.string().optional(),
+    AUTH_CUSTOM_ISSUER: z.string().optional(),
```

Comment:
```suggestion
    AUTH_CUSTOM_ISSUER: z.string().url().optional(),
```

Can we enforce url here as it is used as a hostname later on?

Reply:
Yep that should work, frankly unsure why it wasn't already aligned!